### ISSUE TITLE
Feature Request: Accept height/maxHeight of type 'number'

### ISSUE THREAD

**Issue by islemaster** [AllenFang/react-bootstrap-table#578]
For some reason, the `height` and `maxHeight` properties of `BootstrapTable` only accept strings.  This is documented [in the table's `propTypes`](https://github.com/AllenFang/react-bootstrap-table/blob/master/src/BootstrapTable.js#L907) so it's not a bug, but I was surprised when passing a number [threw an uncaught exception here](https://github.com/AllenFang/react-bootstrap-table/blob/master/src/BootstrapTable.js#L875).

It would be nice if these fields accepted a number, and implicitly treated them as a pixel value [the same way the `style` prop does](https://facebook.github.io/react/tips/style-props-value-px.html).

I want this to work:
```jsx
const MAX_TABLE_HEIGHT = 500;
<BootstrapTable height={MAX_TABLE_HEIGHT}/>
```

Right now I have to do this:
```jsx
const MAX_TABLE_HEIGHT = 500;
<BootstrapTable height={MAX_TABLE_HEIGHT+'px'}/>
```

**Comment by AllenFang** [comment]
sorry for lately reply, busy on work. Thanks @madeinfree, it's well done. I'll merge after reviewing. Thanks :)

**Comment by AllenFang** [comment]
Fixed on ```v2.4.1```.
